src/main/java/com/google/api/codegen/transformer/ApiCallableTransformer.java

```diff
@@ -79,7 +80,10 @@ public ApiCallableTransformer() {
     apiCallableBuilder.memberName(context.getNamer().getSettingsMemberName(method));
     apiCallableBuilder.settingsFunctionName(context.getNamer().getSettingsFunctionName(method));
 
-    if (methodConfig.isBundling()) {
+    if (methodConfig.isGrpcStreaming()) {
+      apiCallableBuilder.type(ApiCallableType.StreamingApiCallable);
+      apiCallableBuilder.streamingType(methodConfig.getGrpcStreaming().getType());
```

Comment:
If this is just gRPC streaming, I'd prefer the view's attribute name to be called `grpcStreamingType` (same for the other places where `streamingType` occurs).